geospaas/nansat_ingestor/managers.py

```diff
@@ -19,39 +19,30 @@
 from geospaas.catalog.models import GeographicLocation, DatasetURI, Source, Dataset
 
 class DatasetManager(models.Manager):
-    default_char_fields = {
-        'entry_id'           : lambda : 'NERSC_' + str(uuid.uuid4()),
-        'entry_title'        : lambda : 'NONE',
-        'summary'            : lambda : 'NONE',
-    }
-
-    default_foreign_keys = {
-        'gcmd_location'      : {'model': Location,
-                                'value': pti.get_gcmd_location('SEA SURFACE')},
-        'data_center'        : {'model': DataCenter,
-                                'value': pti.get_gcmd_provider('NERSC')},
-        'ISO_topic_category' : {'model': ISOTopicCategory,
-                                'value': pti.get_iso19115_topic_category('Oceans')},
-    }
-
-    def get_or_create(self, uri, *args, **kwargs):
+
+    def get_or_create(self, uri, n_points=10, uri_filter_args={}, *args, **kwargs):
```

Comment:
don't initialize `uri_filter_args={}` or it may have a random content.
Initialize as `uri_filter_args=None` and then check inside the method and assign and empty dict